components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIProviderHostObject.java

```diff
@@ -106,6 +109,9 @@ public String getClassName() {
         return "APIProvider";
     }
 
+    // API definitions from swagger v2.0
+    static APIDefinitionFromSwagger20 definitionFromSwagger20 = new APIDefinitionFromSwagger20();
```

Comment:
That was not Intentional and fixed in the commit a7286a4f4a488877478f2f1ea51624b14e179cb7. Thanks for the comment